lib/widgets/Widgets.simba

```diff
@@ -135,12 +135,12 @@ end;
 
 function TReflectWidget.GetScrollX: Integer;
 begin
-  Result := Reflect.Smart.GetFieldInt(Self.Reference, Widget_ScrollX);
+  Result := Reflect.Smart.GetFieldInt(Self.Reference, Widget_RelativeX);
```

Comment:
Yep, takes both to get correct coords: https://github.com/runelite/runelite/blob/master/runelite-api/src/main/java/net/runelite/api/widgets/Widget.java#L213